### DISCUSSION TITLE
time_embed elements location

### DISCUSSION THREAD

**Discussion by miasik** [hako-mikan/sd-webui-supermerger#80] [Q&A] [answered]
The elements below are listed in your table as members of IN00 block.
It was a coincident but I've noticed the the next elements are outside IN00.
They have "Not Merge" in the "Block" column in the "Elements" tab.

> model.diffusion_model.time_embed.0.bias
> model.diffusion_model.time_embed.0.weight
> model.diffusion_model.time_embed.2.bias
> model.diffusion_model.time_embed.2.weight

Where is the truth?

**Comment by miasik** [comment]
Actually, there's no "skip_connection" in IN01

**Comment by hako-mikan** [comment, marked as answer]
Yes. Exactly outside of IN00, but when the merge is processed, it is processed as IN00. This is also true for OUT11. This is also true for MBW, from which this script is derived.

```
                if 'time_embed' in key:
                    weight_index = 0                # before input blocks
                elif '.out.' in key:
                    weight_index = NUM_TOTAL_BLOCKS - 1     # after output blocks
```
This script is quoted from Merge Block Weighted and `time_embed` is considered IN00. 

And thanks to reporting mistake of table.
I fixed table.
